ide/db.sql.editor/src/org/netbeans/modules/db/sql/editor/resources/org-netbeans-modules-db-sql-editor_annotation.xml.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<!DOCTYPE type PUBLIC
+"-//NetBeans//DTD annotation type 1.1//EN"
+"http://www.netbeans.org/dtds/annotation-type-1_1.dtd">
+<type name="org-netbeans-modules-db-sql-editor_annotation"
+      description_key="LAB_db_sql_editor_annotation_warn"
+      localizing_bundle="org.netbeans.modules.db.sql.editor.resources.Bundle"
+      visible="true" 
+      glyph="nbresloc:/org/netbeans/modules/java/resources/error-glyph.gif"
```

Comment:
Please do not use such cross module references - the java cluster could be missing and introducing such a heavy weight dependency will not carry its weight.

Reply:
I was using the glyph icon from the Java cluster.  Is there an alternative common, non Java specific that should be used instead?

Reply:
Copy it into the db.sql.editor module. Done.

Reply:
Will copy over....

However this does beg the question, when searching for that I notice there are 30 copies of this files (not sure if they are all the same but from the thumbnails they look to be)...why do we have to have all these copies?  Why couldn't a "shared resource" folder be used to reduce all these copies?  I thought I remember a discussion about this on the mailing list but maybe I'm misremembering.

Reply:
Actually since this is part of the "annotation" xml file I'm removing this may no longer be needed.

Reply:
For the "why another copy" question: The NetBeans IDE is highly modular. Creating a central icon module would be possible, but at what price? You introduce another level of dependency into the ecosystem. Then the question is, at what level will you bundle? If you create one big pack, questions will be raised why "unused" icons are bundled. To my observation the current decentralized approach works.

There are/were discussions, but the only fallout of this is a set of icons that were redrawn as SVG and are already distributed as part of the IDE.

Reply:
Removed annotation updates so glyoh no longer required